exercises/accumulate/tests/accumulate.rs

```diff
@@ -1,94 +1,86 @@
 extern crate accumulate;
 
-use accumulate::{map_closure, map_function};
+use accumulate::map;
 
 fn square(x: i32) -> i32 {
     x * x
 }
 
-fn abs_val(x: i32) -> i32 {
```

Comment:
I considered asking for this change to be its own commit, but I suppose it's a reasonable thing to roll in since all the lines that reference `abs_val` are already changed in this PR anyway.